.bazelrc

```diff
@@ -17,7 +17,7 @@ build:remote --remote_timeout=7200
 build:linux --config=clang-local
```

Comment:
can you change this to `--config=clang` and give another try?

Reply:
I tested locally and it works, now will try on CI.

Reply:
Sorry for the delay, I got distracted by another issu, but it seems to work.